src/test/java/seedu/address/model/note/NoteTest.java

```diff
@@ -0,0 +1,53 @@
+package seedu.address.model.note;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.tag.Tag;
+
+class NoteTest {
```

Comment:
Should probably test the other methods and constructor as well, but can be done later in a separate PR as its not that important for now.